web/gds-admin-ui/src/components/TopbarSearch.js

```diff
@@ -108,38 +96,77 @@ const MenuList = (props) => {
 
 /* fomates the option label */
 const handleFormatOptionLabel = (option) => {
-    const formattedOption = optionGetter(option);
-    return <div>{formattedOption}</div>;
+    return <Link className='text-muted d-block' to={`/vasps/${option.value}`}>{option.label}</Link>;
```

Comment:
Any reason you're not using `className={classNames('dropdown-item', 'notify-item', 'p-0')}` similar to the items in the example? The class might need an inner span e.g. `<span>{option.label}</span>` but I think that might look better? Or is the `d-block` for overflow? There is also a `text-overflow` class.